keyboards/kingly_keys/ave/staggered/info.json

```
@@ -0,0 +1,119 @@
+{
+  "keyboard_name": "The Ave. (Staggered Ed. PCB)",
+  "url": "",
+  "maintainer": "the-royal",
+  "width": 13.5,
+  "height": 5.5,
+  "layouts": {
+     "LAYOUT_staggered": {
+      "key_count": 51,
+        "layout": [
+         {"label":"k001", "x":0.00, "y":0.00},
+         {"label":"k002", "x":1.50, "y":0.00},
+         {"label":"k011", "x":10.5, "y":0.00},
+         {"label":"k012", "x":11.5, "y":0.00},
+         {"label":"k013", "x":12.5, "y":0.00},
+         {"label":"k101", "x":0.00, "y":1.50},
+         {"label":"k102", "x":1.50, "y":1.50},
+         {"label":"k103", "x":2.50, "y":1.50},
+         {"label":"k104", "x":3.50, "y":1.50},
+         {"label":"k105", "x":4.50, "y":1.50},
+         {"label":"k106", "x":5.50, "y":1.50},
+         {"label":"k107", "x":6.50, "y":1.50},
+         {"label":"k108", "x":7.50, "y":1.50},
+         {"label":"k109", "x":8.50, "y":1.50},
+         {"label":"k110", "x":9.50, "y":1.50},
+         {"label":"k111", "x":10.5, "y":1.50},
+         {"label":"k112", "x":11.5, "y":1.50},
+         {"label":"k113", "x":12.5, "y":1.50},
+         {"label":"k201", "x":0.00, "y":2.50},
+         {"label":"k202", "x":1.50, "y":2.50, "w":1.25},
+         {"label":"k203", "x":2.75, "y":2.50},
+         {"label":"k204", "x":3.75, "y":2.50},
+         {"label":"k205", "x":4.75, "y":2.50},
+         {"label":"k206", "x":5.75, "y":2.50},
+         {"label":"k207", "x":6.75, "y":2.50},
+         {"label":"k208", "x":7.75, "y":2.50},
+         {"label":"k209", "x":8.75, "y":2.50},
+         {"label":"k210", "x":9.75, "y":2.50},
+         {"label":"k211", "x":10.75, "y":2.5},
+         {"label":"k213", "x":11.75, "y":2.5, "w":1.75},
+         {"label":"k301", "x":0.00, "y":3.50},
+         {"label":"k302", "x":1.50, "y":3.50, "w":1.75},
+         {"label":"k303", "x":3.25, "y":3.50},
+         {"label":"k304", "x":4.25, "y":3.50},
+         {"label":"k305", "x":5.25, "y":3.50},
+         {"label":"k306", "x":6.25, "y":3.50},
+         {"label":"k307", "x":7.25, "y":3.50},
+         {"label":"k308", "x":8.25, "y":3.50},
+         {"label":"k309", "x":9.25, "y":3.50},
+         {"label":"k310", "x":10.25, "y":3.50},
+         {"label":"k312", "x":11.25, "y":3.50},
+         {"label":"k313", "x":12.25, "y":3.50, "w":1.25},
+         {"label":"k401", "x":0.00, "y":4.50},
+         {"label":"k402", "x":1.50, "y":4.50, "w":1.25},
+         {"label":"k403", "x":2.75, "y":4.50},
+         {"label":"k404", "x":3.75, "y":4.50, "w":1.25},
+         {"label":"k406", "x":5.00, "y":4.50, "w":2.75},
+         {"label":"k408", "x":7.75, "y":4.50, "w":2.25},
+         {"label":"k411", "x":10.0, "y":4.50, "w":1.25},
+         {"label":"k412", "x":11.0, "y":4.50},
```

```suggestion
         {"label":"k412", "x":11.0, "y":4.75},
```

---

Why should this y value different than the other?

---

Oops, I changed the wrong value. :man_facepalming: 

Should have been:

```suggestion
         {"label":"k412", "x":11.25, "y":4.50},
```